extensions/setname.md

```diff
@@ -0,0 +1,75 @@
+---
+title: IRCv3 `setname` Extension
+layout: spec
+copyrights:
+  -
+    name: "Janne Mareike Koschinski"
+    period: "2019"
+    email: "janne@kuschku.de"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `setname` tag name. Instead, implementations SHOULD use
+the `draft/setname` tag name to be interoperable with other software
+implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed tag name.
+
+## Motivation
+
+Realnames have historically been immutable for a connection, which has
+historically been accepted, as they were rarely used and usually not as present
+in client UIs. Still, multiple IRCds have provided independent, non-standard
+commands to update realnames.
+
+Nowadays, multiple clients show realnames directly in chat, next to the nick
+name, or use information from the realname to provide enhancements, such as
+using emails in the realname to provide gravatar based avatars, leading to
+higher demand for this functionality than ever.
+
+## Description
+
+The `draft/setname` client capability allows clients to change their realname
+(GEICOS) as well as allowing servers to directly inform clients about such a
```

Comment:
think this should be GECOS rather than GEICOS

Reply:
Oops, thanks for finding that typo!